client/src/components/Header_Bar/header.css

```diff
@@ -0,0 +1,67 @@
+.HeadRectangle {
+  position: absolute;
+  width: 375px;
+  height: 57px;
+  left: 0px;
+  top: 0px;
+
+  background: #ffca28;
+}
+
+.textHeader {
```

Comment:
Decide which naming convention you want to use and apply that to all, you've mixed two in here (HeadRectangle, textHeader...)